Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -941,9 +948,7 @@ void RenderBlockFlow::layoutInlineChildren(bool relayoutChildren, LayoutUnit& re
 
 void RenderBlockFlow::performBlockStepSizing(RenderBox& child, LayoutUnit blockStepSizeForChild) const
 {
-    auto& childStyle = child.style();
-
-    if (childStyle.blockStepAlign() != BlockStepAlign::Auto || childStyle.blockStepRound() != BlockStepRound::Up) {
+    if (!BlockStepSizing::childHasSupportedStyle(child.style())) {
         ASSERT_NOT_IMPLEMENTED_YET();
         return;
     }
```

Comment:
This should probably now be:
`ASSERT(BlockStepSizing::childHasSupportedStyle(child.style()))`.

since you can't really reach it otherwise.

Reply:
Good point!